packages/dds/sequence/api-report/sequence.legacy.public.api.md

```diff
@@ -6,6 +6,8 @@
 
 export { BaseSegment }
 
+export { InteriorSequencePlace }
```

Comment:
For reviewers: these public exports aren't real. This is a known API-Extractor limitation. The re-exported APIs here are `@alpha`, so they will only actually be available via the alpha exports.